data/mapcss-mapping.csv

```diff
@@ -93,8 +93,7 @@ highway|primary;27;
 railway|rail;28;
 highway|service|parking_aisle;[highway=service][service=parking_aisle];;name;int_name;29;
 place|hamlet;30;
-# TODO: 50k usages, move
-highway|road;31;
+moved:highway|road:05.2024;31;highway|road
```

Comment:
What is 05.2024 ?

Reply:
date when it was moved/deprecated
so that we know when its OK to re-use it